grapheneexchange/exchange.py

```diff
@@ -873,21 +873,20 @@ def returnTradeHistory(self, currencyPair="all", limit=25):
             filled = self.ws.get_fill_order_history(
                 m["quote"], m["base"], 2 * limit, api="history")
             trades = []
-            for f in filled[1::2] :  # every second entry "fills" the order
+            for f in filled:
                 data = {}
                 data["date"] = f["time"]
                 data["rate"] = self._get_price_filled(f, m)
                 quote = self._get_asset(m["quote"])
-                # base = self._get_asset(m["base"])
-                if f["op"]["pays"]["asset_id"] == m["base"] :
-                    data["type"]   = "buy"
-                    data["amount"] = f["op"]["receives"]["amount"] / 10 ** quote["precision"]
-                else :
-                    data["type"]   = "sell"
-                    data["amount"] = f["op"]["pays"]["amount"] / 10 ** quote["precision"]
-                data["total"]  = data["amount"] * data["rate"]
-                trades.append(data)
-
+                if f["op"]["account_id"] == self.myAccount["id"]:
```

Comment:
I feel maybe I haven't understand correctly while submit the issue? but the fact is that one trade is stored in a pair of members of f["op"], the 2 represent the same trade but each is in perspective of one countparty of the trade(maybe the first one is maker? I am not sure), so first we need to select out the right record with account_id and then process. no need to add else statement as no need to process other records